### PR TITLE
Add balance format util

### PR DESCRIPTION
**Pull request by benxgao**
The PR is to tweak balance format on CENNZnet side, e.g `"123456789" => 12,345.6789`, `12 => 0.0012`.

It would be merged into `implement-balance-format` feature branch, and another PR would be opened to get the util implemented in UI pages. After both PRs merged into feature branch, will get design review then.

### CODE REVIEW

**Comment by alexsednz** [comment]
> _No description provided._

Every PR needs a description. The description should at least include one or more issue number and it should also make it clear if after this change the issue is going to be closed or not.

File: packages/react-components/src/util/toFormatedBalance.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { formatBalance } from '@polkadot/util';
+
+const toFormatedBalance = (value: string): string => {
```
Comment (holygits) [inline packages/react-components/src/util/toFormatedBalance.ts:3]: `POWER` could be a parameter with default value

Comment (holygits) [inline packages/react-components/src/util/toFormatedBalance.ts:3]: spelling nit: `formated` -> `formatted`

Comment (benxgao) [reply packages/react-components/src/util/toFormatedBalance.ts:3]: Definitely!

**Comment by benxgao** [comment]
@holygits It seems `getFormattedBalance.ts` is much similar to polkadot's logic here https://github.com/polkadot-js/common/blob/master/packages/util/src/format/formatBalance.ts#L71 . Just tried to get intuitional approach to solve the problem. If you think the logic in polkadot and `getFormmattedBalance` is more approachable, I do not mind to investigate more. What do you think  @KarishmaBothara ? cc @alexsednz

**Comment by benxgao** [comment]
Later, we could consider situation of balanceInput with type of `BN`, when we really need to.

**Comment by benxgao** [comment]
@holygits  I suppose `1000.1234 ==> 10001234` has been implemented with transaction submission query within BalanceInput; 

Could add a task ticket to add another util function like `asBalance` which may share some common code segments with `toFormattedBalance` to reverse back, if we do need it right now.

Could replace `getFormattedBalance` in following PRs once feature is working. cc @KarishmaBothara

**Pull request merged**